docs/resources/Schema.plist

```diff
@@ -2,6 +2,11 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>#Revision</key>
+	<dict>
+		<key>OpenCore-Version</key>
+		<string>0.9.3</string>
+	</dict>
```

Comment:
Schema docs will need to be updated on a regular cadence following each OpenCore release. Pre-release builds of OpenCore will also need to be smoke-tested against any schema validation in case new schema entries are added that must be specially handled.

Additionally, the usage of the '#Revision' dict mimics the same usage by OCLP in its payload config.plist [here](https://github.com/dortania/OpenCore-Legacy-Patcher/blob/main/payloads/Config/config.plist#L5-L13). Special care must be taken when introducing metadata keys that may conflict with other projects.